plugin.rb

```diff
@@ -191,4 +191,13 @@
 
     false
   end
+
+  register_category_custom_field_type('create_as_qa_default', :boolean)
+  if Site.respond_to? :preloaded_category_custom_fields
+    Site.preloaded_category_custom_fields << 'create_as_qa_default'
+  end
+  add_to_class(:category, :create_as_qa_default) do
+    ActiveModel::Type::Boolean.new.cast(self.custom_fields['create_as_qa_default'])
+  end
+  add_to_serializer(:basic_category, :create_as_qa_default) { object.create_as_qa_default }
```

Comment:
Can we add a test to verify that the correct value is present in the serializer?

Reply:
Also added! https://github.com/discourse/discourse-question-answer/pull/96/commits/f13168aef0a29ad63bc2b472ea6ce787371787c6